password/impl/src/main/java/org/wildfly/security/password/impl/PasswordUtil.java

```diff
@@ -35,7 +37,24 @@ final class PasswordUtil {
      */
     public static byte[] generateRandomSalt(int saltSize) {
         byte[] randomSalt = new byte[saltSize];
-        ThreadLocalRandom.current().nextBytes(randomSalt);
+        RANDOM.nextBytes(randomSalt);
         return randomSalt;
     }
+
+    /**
+     * Generate a random salt as int.
+     *
+     * @return a byte array representing the random salt
+     */
+    public static int generateRandomSaltInt() {
```

Comment:
I know the original was public but maybe reduce the visibility here as only used within this package.